src/skin/frontend/base/default/omise/gateway/css/style.css

```diff
@@ -97,3 +97,22 @@
 .banks-list .omise-logo-wrapper > div.logo-ktc {
     background: url('../images/ktc.svg') #ffffff;
 }
+
+.omise-test-warning {
```

Comment:
I would use existing class.. so if somebody has some theme installed it will look nice...

Reply:
I disagree. It needs to look as alien and visible as possible, so they won't miss it. Looking nice is not required